src/components/search/search.view.js

```diff
@@ -10,10 +10,15 @@ import {h} from '@cycle/dom'
 import css from './search.style'
 
 const resetValue = () =>
-  h(`input.${css.input}`, {type: 'text', placeholder: 'Search', value: ''})
+  h(`input.${css.input}`, {
+    attrs: {type: 'text', placeholder: 'Search'},
+    props: {value: ''}
+  })
 
 const ignoreValue = () =>
-  h(`input.${css.input}`, {type: 'text', placeholder: 'Search'})
+  h(`input.${css.input}`, {
+    attrs: {type: 'text', placeholder: 'Search'}
+  })
```

Comment:
can you create one single function and logically control if `value` prop should be provided or not.